.github/workflows/test.yml

```diff
@@ -222,15 +222,14 @@ jobs:
       continue-on-error: true
       env:
         COVERALLS_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-        GO111MODULE: off
       run: |
         set -x
-        go get github.com/mattn/goveralls
-        go get github.com/modocache/gover
+        go install github.com/mattn/goveralls@latest
+        go install github.com/modocache/gover@latest
         PATH=$PATH:$(go env GOPATH)/bin
 
-        mkdir -p $(go env GOPATH)/src/github.com/ovn-kubernetes
-        ln -sf $(pwd) $(go env GOPATH)/src/github.com/ovn-kubernetes/ovn-kubernetes
+        mkdir -p $(go env GOPATH)/src/github.com/ovn-org
```

Comment:
is this just a `ovn-kubernetes->ovn-org` cosmetic change ?

seeing this action is still failing.

```
++ pwd
++ go env GOPATH
+ ln -sf /home/runner/work/ovn-kubernetes/ovn-kubernetes /home/runner/go/src/github.com/ovn-org/ovn-kubernetes
+ gover
+ goveralls -coverprofile=gover.coverprofile -service=github
cannot find file "github.com/ovn-org/ovn-kubernetes/go-controller/hybrid-overlay/pkg/controller/ho_node_linux.go": can't find "ho_node_linux.go": no required module provides package github.com/ovn-org/ovn-kubernetes/go-controller/hybrid-overlay/pkg/controller: go.mod file not found in current directory or any parent directory; see 'go help modules'
```

Reply:
I know. Working on it.